<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SLR_GUIClass</class>
 <widget class="QMainWindow" name="SLR_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SLR_GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SLR_GUI.qrc">
    <normaloff>:/icons/res/analyzser.ico</normaloff>:/icons/res/analyzser.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnOpen">
            <property name="text">
             <string>打开文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSave">
            <property name="text">
             <string>保存文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAnalyse">
            <property name="text">
             <string>开始分析</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>语法输入框</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="inputEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lblFirst">
            <property name="text">
             <string>First集合</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableFirst"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblFollow">
            <property name="text">
             <string>Follow集合</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableFollow"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lblLR">
          <property name="text">
           <string>LR(0)DFA图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableDFA"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="lblSLR">
          <property name="text">
           <string>SLR(1)分析表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableSLR"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="lblNotSLR">
          <property name="text">
           <string>不是SLR原因</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textReason"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SLR_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
